packages/Sandblocks-Core/SBBlock.class.st

```diff
@@ -16,6 +16,24 @@ Class {
 	#category : #'Sandblocks-Core'
 }
 
+{ #category : #'as yet unclassified' }
+SBBlock class >> defaultRepelStrength [
+
+	^ 5000.0
+]
+
+{ #category : #'as yet unclassified' }
+SBBlock class >> implicitAttractionStrength [
+
+	^ 0.25
+]
+
+{ #category : #'as yet unclassified' }
+SBBlock class >> maxRepelDistance [
+
+	^ 100
+]
```

Comment:
We removed them for now, because w/o a presentation backing it up, we think its not that useful. Having an editor inside another also was kinda broken. I guess we could add some more comments inside the examples if you think that'd be beneficial.

Reply:
unles it's A LOT of code, I think I'd be fine with keeping it in there, even if it's a little buggy. I think it's a nice showcase that may be worth fixing later on.